finetune/model.py

```diff
@@ -296,7 +286,7 @@ def clip_gradients(grads_n_vars):
                 learning_rate_decay_fn=lr_decay,
                 increment_global_step=True,
                 summaries=summaries,
-                colocate_gradients_with_ops=False,
+                colocate_gradients_with_ops=True,
```

Comment:
Should this always be on?  Or rather -- is always having this on fine?

Reply:
This was previously on, It was a regression introduced in an earlier commit that changed this.